composer.json

```diff
@@ -1,6 +1,25 @@
 {
+	"name": "phpbb-de/phpbb-translation",
+	"description": "German Translation of phpBB",
+	"type": "phpbb-language",
+	"homepage": "https://github.com/phpbb-de/phpbb-translation",
+	"version": "3.1.11-dev",
```

Comment:
Die Angabe in der Readme hatte ich noch immer als ganz brauchbar gehalten, um mal auf einen Blick zu sehen für welche Version das Paket so ist, wenn es Probleme gibt.